duckduckgo/iOS
# Feature/bunn/download manager

Task/Issue URL: https://app.asana.com/0/1201188456623839/1201600067271023/f
Tech Design URL: https://app.asana.com/0/481882893211075/1201601635648875/f
CC:

**Description**:
Add DownloadsManager to the project. Note, this PR is not to me merged on develop, but on feature/downloads

**Steps to test this PR**:
1. Tap on a Wallet item like [this one](https://www.passsource.com/pass/register.php?hashedSerialNumber=eNortjIysVIKcAmuKPD3Ci8KDyjMCwzwT8-2qCwOtLVVsgZcMKbZCjU,&psRegisteredFlag=1&psMessage=&) and see if the browser presents the wallet pass
2. Tap on downloads like this [example](https://www.thinkbroadband.com/download) and check the logs to see if the download is working (Also check tmp and documents folder to see the download)
3. Tap on multiple downloads with the same name to see if the app correctly appends a unique number to the file
4. Run tests and see if they are passing


**Device Testing**:

* [x] iPad Pro

**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
